src/pages/Swap/index.tsx

```diff
@@ -107,16 +114,16 @@ export default function Swap({ history }: RouteComponentProps) {
 
   // swap state
   const { independentField, typedValue, recipient } = useSwapState()
+  const newLocal = useDerivedSwapInfo(toggledVersion)
   const {
-    v2Trade,
-    v3TradeState: { trade: v3Trade, state: v3TradeState },
-    toggledTrade: trade,
+    v3Trade: { state: v3TradeState },
```

Comment:
i think ideally we push this into the hook, and expose a single loading state for whether v2/v3 trades are loaded/current/syncing. maybe this will help clean up/prevent the inconsistent loading state issues in the component